### ISSUE TITLE
Error: missing keyspace in migration statement

### ISSUE THREAD

**Issue by holgerw** [Cobliteam/cassandra-migrate#5]
I have a migration file, referencing its table without a keyspace. The migration fails, with the following error message:

```bash
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/site-packages/cassandra_migrate/migrator.py", line 380, in _apply_migration
InvalidRequest: Error from server: code=2200 [Invalid query] message="No keyspace has been specified. USE a keyspace, or explicitly specify keyspace.tablename"
```

cassandra-migrate could issue a USE statement before the first migration, considering the keyspace configured in its configuration file. If we hardcode the schema in every migration we loose the ability to switch between dev and prod keyspaces.

I really appreciate cassandra-migrate, and am trying to establish it at my company. Unfortunately I am not experienced in Python, to submit a pull request, instead of opening the issues.

**Comment by danielkza** [comment]
Thank you for the report! I will make sure the default keyspace is set in the session.

> I really appreciate cassandra-migrate, and am trying to establish it at my company. Unfortunately I am not experienced in Python, to submit a pull request, instead of opening the issues.

That's not a problem at all, testing is already very valuable by itself :)

**Comment by revolaution** [comment]
We're also running into this issue and would really appreciate any updates on this! We like the YAML setup and would like to define the keyspaces there for our migrations.

**Comment by revolaution** [comment]
Thanks a lot!

**Comment by danielkza** [comment]
Should be fixed in 0.2.1. Haven't had the chance to test it that much, so please do if you can!

**Comment by holgerw** [comment]
This issue was blocking our adoption at NIC.br . Started using cassandra-migrate with version 0.2.2. Thank's for your efforts.
